engine/games/slot_machine.asm

```diff
@@ -15,17 +15,21 @@ DEF SLOTS_NO_BIAS EQU -1
 DEF REEL_SIZE EQU 15
 
 ; Constants for slot_reel offsets (see macros/wram.asm)
-DEF REEL_ACTION        EQUS "(wReel1ReelAction - wReel1)"
-DEF REEL_TILEMAP_ADDR  EQUS "(wReel1TilemapAddr - wReel1)"
-DEF REEL_POSITION      EQUS "(wReel1Position - wReel1)"
-DEF REEL_SPIN_DISTANCE EQUS "(wReel1SpinDistance - wReel1)"
-DEF REEL_SPIN_RATE     EQUS "(wReel1SpinRate - wReel1)"
-DEF REEL_OAM_ADDR      EQUS "(wReel1OAMAddr - wReel1)"
-DEF REEL_X_COORD       EQUS "(wReel1XCoord - wReel1)"
-DEF REEL_MANIP_COUNTER EQUS "(wReel1ManipCounter - wReel1)"
-DEF REEL_MANIP_DELAY   EQUS "(wReel1ManipDelay - wReel1)"
-DEF REEL_FIELD_0B      EQUS "(wReel1Field0b - wReel1)"
-DEF REEL_STOP_DELAY    EQUS "(wReel1StopDelay - wReel1)"
+rsreset
+DEF REEL_ACTION        rb
+DEF REEL_TILEMAP_ADDR  rw
+DEF REEL_POSITION      rb
+DEF REEL_SPIN_DISTANCE rb
+DEF REEL_SPIN_RATE     rb
+DEF REEL_OAM_ADDR      rw
+DEF REEL_X_COORD       rb
+DEF REEL_MANIP_COUNTER rb
+DEF REEL_MANIP_DELAY   rb
+DEF REEL_FIELD_0B      rb
+DEF REEL_FIELD_0C      rb
+DEF REEL_FIELD_0D      rb
+DEF REEL_FIELD_0E      rb
```

Comment:
```suggestion
                       rb_skip 4
```

Reply:
REEL_FIELD_0B is used; So I left it in.

Reply:
Looks like it's only used for the countdown after starting the `REEL_ACTION_WAIT_DROP_REEL` action.

```diff
ReelAction_WaitDropReel:
-	ld hl, REEL_FIELD_0B
+	ld hl, REEL_DROP_COUNTER
	add hl, bc
	ld a, [hl]
	and a
	jr z, .DropReel
	dec [hl]
	ret

.DropReel:
	ld hl, REEL_ACTION
	add hl, bc
-	dec [hl]
+	dec [hl] ; REEL_ACTION_CHECK_DROP_REEL
	ld hl, REEL_SPIN_RATE
	add hl, bc
	ld [hl], 8
	ret
```